src/Demo.Api/Mapper/DemoProfile.cs

```diff
@@ -8,8 +8,12 @@ public class DemoProfile : Profile
     {
         public DemoProfile()
         {
-            CreateMap<ProductModel, ProductApiModel>().ReverseMap();
-            CreateMap<CategoryModel, CategoryApiModel>().ReverseMap();
+            CreateMap<CategoryModel, CategoryCreateRequest>().ReverseMap();
```

Comment:
Request and Response both are mapping to same CategoryModel?

Reply:
@barotbm reason is we have one business model. Here idea is to have separate api model for in coming request to eliminate unnecessary properties on request. For example, POST category doesn't need CategoryId since it's not created yet but PUT category request needs Category Id.